### PR TITLE
release notes for 1.1.0-M1

### PR DESCRIPTION
**Pull request by pjfanning**
RC vote has passed

### CODE REVIEW

File: docs/src/main/paradox/release-notes/releases-1.1.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+As with all milestone releases, this release is not recommended for production use - it is designed to allow users to try out the changes in a test environment.
+
+### Bug Fixes
+* When PreparedStatement initialization fails, the code used not recover ([#103](https://github.com/apache/pekko-persistence-cassandra/issues/103))
+
+### Other Changes
+* Improve the pattern matching for CassandraJournal options ([PR28](https://github.com/apache/pekko-persistence-cassandra/pull/28))
+
+### Dependency Upgrades
+* Apache assandra Driver 4.18 (replaces the Datastax Driver that was used before, this was donated to Apache Cassandra and they released 4.18 from this)
```
Comment (jxnu-liguobin) [inline docs/src/main/paradox/release-notes/releases-1.1.md:14]: Is there a missing C？

Comment (pjfanning) [reply docs/src/main/paradox/release-notes/releases-1.1.md]: Thanks - I've committed a fix

File: docs/src/main/paradox/release-notes/releases-1.1.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Release Notes (1.1.x)
+
+## 1.1.0-M1
+Release notes for Apache Pekko Persistence Cassandra 1.1.0-M1. See [GitHub Milestone](https://github.com/apache/pekko-persistence-cassandra/milestone/1?closed=1) for a fuller list of changes.
+As with all milestone releases, this release is not recommended for production use - it is designed to allow users to try out the changes in a test environment.
+
+### Bug Fixes
+* When PreparedStatement initialization fails, the code used not recover ([#103](https://github.com/apache/pekko-persistence-cassandra/issues/103))
```
Comment (nvollmar) [inline docs/src/main/paradox/release-notes/releases-1.1.md:8]: used not to recover

**Review by nvollmar — APPROVED** [review]
lgtm

File: docs/src/main/paradox/release-notes/releases-1.1.md (hunk 2 above)
Comment (pjfanning) [reply docs/src/main/paradox/release-notes/releases-1.1.md]: changed

**Review by nvollmar — APPROVED** [review]

**Pull request merged**